### PR TITLE
Feature findAll and findById

### PR DESCRIPTION
**Pull request by Robert-Michalski**

### CODE REVIEW

File: src/main/java/com/ormanager/orm/mapper/ObjectMapper.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                } else if (field.getType() == Long.class) {
+                    field.set(t, resultSet.getLong(fieldName));
+                } else if (field.getType() == String.class) {
+                    field.set(t, resultSet.getString(fieldName));
+                } else if (field.getType() == LocalDate.class) {
+                    field.set(t, resultSet.getDate(fieldName).toLocalDate());
+                }
+            }
+        } catch (IllegalAccessException | SQLException e) {
+            e.printStackTrace();
```
Comment (olhakad) [inline src/main/java/com/ormanager/orm/mapper/ObjectMapper.java]: please don't use print stack trace, use logger

File: src/main/java/com/ormanager/orm/OrmManager.java [hunk 2]
```diff
@@ -1,65 +1,153 @@
...
+    }
+
+    public List<Field> getAllColumnsButId(T t) {
+        return Arrays.stream(t.getClass().getDeclaredFields())
+                .filter(v -> !v.isAnnotationPresent(Id.class))
+                .collect(Collectors.toList());
+    }
+
+    public <T> Optional<T> findById(Serializable id, Class<T> cls) {
+        T t = null;
```
Comment (olhakad) [inline src/main/java/com/ormanager/orm/OrmManager.java:111]: T can be defined inside catch block, no need to set it here to null

**Review by olhakad — CHANGES_REQUESTED** [review]

**Comment by olhakad** [comment]
Please resolve conflicts

File: src/main/java/com/ormanager/orm/OrmManager.java (hunk 2 above)
Comment (bogdansoft) [reply src/main/java/com/ormanager/orm/OrmManager.java:111]: It return Optional.ofNullable(t) in the end

**Review by bogdansoft — APPROVED** [review]

**Review by bogdansoft — DISMISSED** [review]

**Review by olhakad — APPROVED** [review]

**Pull request merged**